### PR TITLE
Add goimports check to nogo linter

### PR DESCRIPTION
**Pull request by kaweue**
This PR fixes imports order and enables automatic check of it during the build.

### CODE REVIEW

**Comment by kaweue** [comment]
recheck ha

File: statusmonitor/config_status_monitor_test.go [hunk 1]
```diff
@@ -3,10 +3,12 @@ package main
...
-	contrailOperatorTypes "github.com/Juniper/contrail-operator/pkg/apis/contrail/v1alpha1"
-	"github.com/stretchr/testify/assert"
 	"io/ioutil"
 
+	"github.com/stretchr/testify/assert"
+
+	contrailOperatorTypes "github.com/Juniper/contrail-operator/pkg/apis/contrail/v1alpha1"
+
 	"net/http"
 	"testing"
```
Comment (mfigurski) [inline statusmonitor/config_status_monitor_test.go:13]: These two should be in first section.

File: statusmonitor/main.go [hunk 2]
```diff
@@ -24,8 +24,9 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/client-go/util/retry"
```
Comment (mfigurski) [inline statusmonitor/main.go:26]: Unnecessary empty line.

File: test/env/client/postgres.go [hunk 3]
```diff
@@ -3,6 +3,7 @@ package client
 import (
 	"context"
 	"fmt"
+
 	"github.com/go-pg/pg/v10/orm"
```
Comment (mfigurski) [inline test/env/client/postgres.go:8]: Unnecessary empty line.

File: test/wait/contrail.go [hunk 4]
```diff
@@ -5,9 +5,10 @@ import (
...
 	"time"
 
+	"github.com/operator-framework/operator-sdk/pkg/test"
+
 	contrail "github.com/Juniper/contrail-operator/pkg/apis/contrail/v1alpha1"
 	"github.com/Juniper/contrail-operator/test/logger"
-	"github.com/operator-framework/operator-sdk/pkg/test"
 
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (mfigurski) [inline test/wait/contrail.go:14]: These ones should be one section higher.

**Review by mfigurski — APPROVED** [review]

File: pkg/controller/config/config_controller.go [hunk 5]
```diff
@@ -20,6 +19,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 
+	"github.com/Juniper/contrail-operator/pkg/controller/utils"
```
Comment (Dysproz) [inline pkg/controller/config/config_controller.go:22]: Shouldn't all imports from contrail-operator be in one block?

Comment (kaweue) [reply pkg/controller/config/config_controller.go:22]: This was fixed automatically by goimports

File: test/wait/contrail.go (hunk 4 above)
Comment (kaweue) [reply test/wait/contrail.go:10]: fixed

File: test/env/client/postgres.go (hunk 3 above)
Comment (kaweue) [reply test/env/client/postgres.go]: fixed

File: statusmonitor/main.go (hunk 2 above)
Comment (kaweue) [reply statusmonitor/main.go:27]: fixed

File: pkg/controller/config/config_controller.go (hunk 5 above)
Comment (kaweue) [reply pkg/controller/config/config_controller.go:25]: done

**Review by Dysproz — APPROVED** [review]

**Pull request merged**